barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/verifier.test.cpp

```diff
@@ -141,30 +161,33 @@ template <typename OuterFlavor> class HonkRecursiveVerifierTest : public testing
         EXPECT_EQ(verifier.key->q_r.get_value(), verification_key->q_r);
```

Comment:
I did this kind of hacky one off check on polys because at the time it wasn't easy to loop over things. Can we update this to use the appropriate flavor style getter and a loop to check all polys? Not a big deal if that's not possible for some reason but I think it is. I think that will also avoid the need for the conditional for goblin

Reply:
thanks!